plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -165,15 +165,19 @@ func defaultPredicates() sets.String {
 
 		// Fit is determined based on whether a pod can tolerate all of the node's taints
 		factory.RegisterFitPredicate("PodToleratesNodeTaints", predicates.PodToleratesNodeTaints),
+	)
 
-		// Fit is determined by volume zone requirements.
-		factory.RegisterFitPredicateFactory(
-			"NoVolumeNodeConflict",
+	if utilfeature.DefaultFeatureGate.Enabled(features.VolumeScheduling) {
```

Comment:
@bsalamat is this an acceptable way to add new predicates under an alpha feature?

Reply:
Ah actually, this is not working.  The predicates list is initialized during init(), which seems to be before the point that the feature gates are set.